TimedArithmeticQuiz2/src/App.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { Quiz } from "./components/Quiz";
+import { QuizCreationForm } from "./components/QuizCreationForm";
+
+
+function App() {
+  const [quizDataList, setQuizDataList] = useState([]);
+  
+  function createQuiz(formData) {
+      setQuizDataList(list => [...list, formData])
+  }
+  
+  const quizList = quizDataList.map(quizData => {
+    return <Quiz 
+      key={quizData.id} 
+      numberRange={[quizData.minLimit,quizData.maxLimit]} 
```

Comment:
Rather than passing each data argument separately, we can just pass the quiz object and destructure.